src/sentry/models/options/option.py

```diff
@@ -20,6 +34,9 @@ class Option(Model):  # type: ignore
     key = models.CharField(max_length=128, unique=True)
     value = PickledObjectField()
     last_updated = models.DateTimeField(default=timezone.now)
+    last_updated_by = models.CharField(
+        max_length=128, choices=OptionsTypes.choices(), default="legacy"
+    )
```

Comment:
Will we need to query by this at all?

Reply:
I don't anticipate writing queries against this column directly. This is specifically for my config options tooling that wants to know who set the most recent value. I can't think of a useful instance where we need to query by the last_update_by column. If we did, there are only ~300 options entries so it's a pretty small table all things considered.